src/Sarif/Autogenerated/AnnotatedCodeLocation.cs

```diff
@@ -36,7 +36,9 @@ public SarifNodeKind SarifNodeKind
         /// An identifier for the location, unique within the scope of the code flow within which it occurs.
         /// </summary>
         [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false)]
-        public string Id { get; set; }
+        [Newtonsoft.Json.JsonConverter(typeof(Microsoft.CodeAnalysis.Sarif.Readers.AnnotatedCodeLocationIdConverter))]
+        [Newtonsoft.Json.JsonProperty("id", DefaultValueHandling = Newtonsoft.Json.DefaultValueHandling.IgnoreAndPopulate)]
```

Comment:
Here are the two attributes needed to properly handle the Id property. We specify the converter, and we ensure that the converter is called even when the incoming value is the default (0). This is what required us to upgrade to a new JSchema version.